scripts/nonVerboseFlagsInGitHubCIAndScripts.fsx

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env -S dotnet fsi
+
+open System
+open System.IO
+
+#load "../src/FileConventions/Library.fs"
+#load "../src/FileConventions/Helpers.fs"
+
+let rootDir = Path.Combine(__SOURCE_DIRECTORY__, "..") |> DirectoryInfo
+
+let validExtensions =
+    seq {
+        ".yml"
+        ".fsx"
+        ".fs"
+        ".sh"
+    }
+
+let invalidFiles =
+    validExtensions
+    |> Seq.map(fun ext ->
+        Helpers.GetInvalidFiles
+            rootDir
+            ("*" + ext)
```

Comment:
@realmarv aren't you missing a dot after * here?

Reply:
No ext already has a dot